src/os/host-start.r

```diff
@@ -423,6 +421,8 @@ host-start: function [
         ]
     ]
 
+    boot-embedded: get-encap system/options/boot
+
```

Comment:
What we should do is make sure that whatever process is needed to get the real path, we make sure that gets passed in as argv/1 to the host-start...so that system/options/boot will be guaranteed as the right path...